workers/unity/Packages/com.improbable.gdk.core/Components/WorkerEntityComponents.cs

```diff
@@ -3,25 +3,31 @@
 namespace Improbable.Gdk.Core
 {
     /// <summary>
-    ///     Component denoting a worker entity
+    ///     ECS Component denoting a worker entity
     /// </summary>
     public struct WorkerEntityTag : IComponentData
     {
     }
 
     /// <summary>
-    ///     Component added to the worker entity immediately after establishing a connection to a SpatialOS deployment
-    ///     Removed at the end of the tick it was added
+    ///     Component added to the worker entity immediately after establishing a connection to a SpatialOS deployment.
```

Comment:
add ECS here as well?

Reply:
You and @samiwh have differing opinions on this 😛 

I'm generally in favor of adding the explicit type of component

Reply:
seeing as we have GameObject components, SpatialOS components and ECS components.... let's make it explicit :P